modules/javafx.graphics/src/main/java/com/sun/javafx/scene/CameraHelper.java

```diff
@@ -57,7 +57,7 @@ public static void initHelper(Camera camera) {
 
     @Override
     protected NGNode createPeerImpl(Node node) {
-        throw new UnsupportedOperationException("Applications should not extend the Camera class directly.");
+        throw new AssertionError();
```

Comment:
is this the right error type?
why change UnsupportedOperationException?

Reply:
When `Camera` is sealed, this method cannot be called by user code. `UnsupportedOperationException` would be okay if it could reasonably be called, but this is now an invariant that would signal a bug in JavaFX.

Reply:
maybe an explanatory comment should be added?

Reply:
We typically use `InternalError` for the case where there should be no possible reason for a particular case, but also use `AssertionError` in some cases. Either is OK with me. I agree that a comment to the affect of "Should not ever get here" wouldn't be a bad idea.

Reply:
I've added a comment to `CameraHelper` and `LightBaseHelper`.